batch/batch/driver/templates/index.html

```diff
@@ -48,12 +48,12 @@ <h2>Pools</h2>
       <td><a href="{{ base_path }}/inst_coll/pool/{{ pool.name }}">{{ pool.name }}</a></td>
       <td>{{ pool.worker_type }}</td>
       <td>{{ pool.preemptible }}</td>
-      <td class="numeric-cell">{{ pool.n_instances_by_state['pending'] }}</td>
-      <td class="numeric-cell">{{ pool.n_instances_by_state['active'] }}</td>
-      <td class="numeric-cell">{{ pool.n_instances_by_state['inactive'] }}</td>
-      <td class="numeric-cell">{{ pool.n_instances_by_state['deleted'] }}</td>
-      <td class="numeric-cell">{{ pool.live_total_cores_mcpu / 1000 }}</td>
-      <td class="numeric-cell">{{ pool.live_free_cores_mcpu / 1000 }}</td>
+      <td class="numeric-cell">{{ pool.current_worker_version_stats.n_instances_by_state['pending'] }}</td>
+      <td class="numeric-cell">{{ pool.current_worker_version_stats.n_instances_by_state['active'] }}</td>
+      <td class="numeric-cell">{{ pool.current_worker_version_stats.n_instances_by_state['inactive'] }}</td>
+      <td class="numeric-cell">{{ pool.current_worker_version_stats.n_instances_by_state['deleted'] }}</td>
+      <td class="numeric-cell">{{ pool.current_worker_version_stats.live_total_cores_mcpu / 1000 }}</td>
+      <td class="numeric-cell">{{ pool.current_worker_version_stats.live_free_cores_mcpu / 1000 }}</td>
```

Comment:
Hmm. I'm a bit worried that this isn't clearly communicated in the UI. I think we should include the total over all versions somewhere.

Relatedly, if its simple to modify while we're here, it has *long* bothered me that we include the "activating" cores in the "live total". I feel like I want three notions:
- `{{free}} / {{total}}` for "schedule-able"
- `{{ total }}` "activating"
- `{{ total }}` not-inactive (so, anything we're paying for)